Rubberduck.VBEditor.VB6/SafeComWrappers/VB/VBE.cs

```diff
@@ -29,11 +29,42 @@ public VBE(VB.VBE target, bool rewrapping = false)
         public ICodePane ActiveCodePane
         {
             get => new CodePane(IsWrappingNullReference ? null : Target.ActiveCodePane);
-            set
+            set => SetActiveCodePane(value);
+        }
+
+        private void SetActiveCodePane(ICodePane codePane)
+        {
+            if (IsWrappingNullReference || !(codePane is CodePane pane))
             {
-                if (!IsWrappingNullReference)
+                return;
+            }
+
+            Target.ActiveCodePane = pane.Target;
+            ForceFocus(codePane);
+        }
+
+        private void ForceFocus(ICodePane codePane)
```

Comment:
Can you do me a favor and drop a TODO on this? The subclass manager should have all of the window handles associated with code panes now - I can probably do this without using FindChildWindow at this point.

Reply:
Sure, I can add a todo. Just one question, is the subclass manager in an assembly we can reference from here?

Reply:
The [subclass manager](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.VBEEditor/WindowsApi/SubclassManager.cs) is in the `Rubberduck.VBEEditor` assembly; you should be able to reference it from VBA/VB6 assembly.